### PR TITLE
Testdiff

### PR DESCRIPTION
**Pull request by itsAlexNguyen**
This is an example PR.

### CODE REVIEW

**Comment by ReviewerOfCode** [comment]
Testing comments

**Comment by ReviewerOfCode** [comment]
I generated a SimpleDiff here -> https://tranquil-earth-41184.herokuapp.com/diff?branch=master

**Comment by ReviewerOfCode** [comment]
I generated a SimpleDiff here -> https://tranquil-earth-41184.herokuapp.com/diff?branch=testdiff

**Comment by ReviewerOfCode** [comment]
I generated a SimpleDiff here -> https://tranquil-earth-41184.herokuapp.com/diff?branch=testdiff

**Comment by ReviewerOfCode** [comment]
I generated a SimpleDiff here -> https://tranquil-earth-41184.herokuapp.com/diff?branch=testdiff

**Comment by ReviewerOfCode** [comment]
I generated a SimpleDiff here -> https://tranquil-earth-41184.herokuapp.com/diff?branch=testdiff

**Comment by ReviewerOfCode** [comment]
I generated a SimpleDiff here -> https://tranquil-earth-41184.herokuapp.com/diff?branch=testdiff

**Comment by ReviewerOfCode** [comment]
I generated a SimpleDiff here -> https://tranquil-earth-41184.herokuapp.com/diff?branch=testdiff

File: Example_v0.java [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-
 public class A {
-    private static int Z;
+    private static int X;
+    private static int Y = 5;
 
-    public A() {
-        A.Z = 5;
+    public A(){
+        A.X = 5;
```
Comment (itsAlexNguyen) [inline Example_v0.java:6]: What does this change do?

**Pull request closed without merging**